### PR TITLE
Blog post introducing the IPFS Mobile Design guidelines

### PR DESCRIPTION
**Pull request by jkosem**
^^ @autonome

### CODE REVIEW

File: content/post/ipfs-mobile-design-research [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+date: 2020-03-30
```
Comment (momack2) [inline content/post/ipfs-mobile-design-research:2]: just a reminder to update before posting =]

File: content/post/ipfs-mobile-design-research.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+date: 2020-03-30
+url: 2020-03-30-ipfs-mobile-design-research
+title: IPFS Mobile Design Research
+author: Jim Kosem, Dietrich Ayala
+tags: mobile-design-guidelines, Mobile, Design, Research
+---
+
+The focus of development of IPFS to date has been primarily on desktop and server class hardware. However, the growth of the internet for more than a decade has been almost entirely on mobile devices.
+
+This growth at the furthest edge of the network requires solutions at that edge. As long as the IPFS network remains inaccessable on mobile devices, the value of the network where it matters most is not realized - IPFS will be a step removed from where end users are, its features restricted and stoppable, and its offline use-cases constrained by the application development patterns of centralized networks and services.
```
Comment (momack2) [inline content/post/ipfs-mobile-design-research.md:11]: IPFS now has default support on Opera for Android - so I don't think it's reasonable to see the IPFS networks "remains inaccessible on mobile devices" -- however I think you could flip this around and state that mobile is a huge opportunity for the IPFS network that requires future development

File: content/post/ipfs-mobile-design-research (hunk 1 above)
Comment (jkosem) [reply content/post/ipfs-mobile-design-research]: Changed!

File: content/post/ipfs-mobile-design-research.md (hunk 2 above)
Comment (jkosem) [reply content/post/ipfs-mobile-design-research.md]: @momack2 Fixed. Sorry, looks like the Opera for Android came out after this was written, but it is in there now and now the paragraph is framed in terms of taking the problem and solving user needs and leading to more adoption.

**Pull request merged**